bin/cli.js

```diff
@@ -50,6 +50,11 @@ const cliFlags = yargs
     'describe': 'Path to config file',
     'type': 'string'
   })
+  .option('share-with-timeline-viewer', {
```

Comment:
What if we use `--upload` just for uploading stuff to gdrive and `--share` for sharing?
Then it will be bit hard for user to right `pwmetrics http://example.com --upload --share`, but from developers perspective of view it will be clean, no magic under `--upload` flag though...